2025/timed-text-wg.html

```diff
@@ -227,7 +227,7 @@ <h3>New Normative Specifications</h3>
 
   </dl>
 
-  <h3>New Normative Specifications Continuously Under Development</h3>
+  <h3>Normative Specifications Continuously Under Development</h3>
```

Comment:
This section does include permission for new normative specifications too, so I suggest:

```suggestion
  <h3>New or Continuing Normative Specifications</h3>
```

Reply:
ah, rather than this, how about to move the one line into section above? (should be simpler...)

Reply:
Yes that's a much better idea.

Reply:
@nigelmegitt updated, please check
/cc @gkatsev